src/browser/ipc/event/AuthenticationEvents.ts

```diff
@@ -0,0 +1,155 @@
+import { IPCEventRegisterBase } from '../../../common/ipc/BaseIPC'
```

Comment:
It's a bit annoying that we will lose track of history because of this refactor.